parachain/pallets/commitments/src/merkle_tree.rs

```diff
@@ -0,0 +1,62 @@
+use codec::Encode;
+use sp_core::{Hasher, H256};
+use sp_std::prelude::*;
+use sp_trie::TrieConfiguration;
+
+use hash256_std_hasher::Hash256StdHasher;
```

Comment:
hmm, you should be using this hasher if possible, its already provided to the commitments pallet: https://github.com/Snowfork/polkadot-ethereum/blob/main/parachain/pallets/commitments/src/lib.rs#L55

Its a keccak256 hasher so is compatible.

Reply:
I'll try to make it work with that one